vnode_controller/vnode_controller.go

```
@@ -463,53 +489,40 @@
 		// Start a new goroutine to run the VNode
 		go vn.Run(vnCtx, initData)
 
 		// If the VNode is not ready after a minute, log an error and cancel the context
 		if err = vn.WaitReady(vnCtx, time.Minute); err != nil {
 			err = errpkg.Wrap(err, "Error waiting vnode ready")
+			vNodeController.vNodeStore.DeleteVNode(nodeName)
+			log.G(vnCtx).Infof("node exit %s", nodeName)
 			vnCancel()
 			return
+		} else {
+			vNodeController.vNodeStore.AddVNode(nodeName, vn)
```

golangci-lint (1.61.0)

Error return value of `vNodeController.vNodeStore.AddVNode` is not checked

(errcheck)